src/page-modules/assistant/server/journey-planner/index.ts

```diff
@@ -230,6 +231,7 @@ export function createJourneyApi(
         expectedStartTime: singleTripPattern?.expectedStartTime,
         expectedEndTime: singleTripPattern?.expectedEndTime,
         duration: singleTripPattern?.duration ?? 0,
+        walkDistance: singleTripPattern?.streetDistance ?? 0,
```

Comment:
Should we rename this to streetDistance internally as well?

Reply:
walkDistance makes sense though.

Reply:
We could do that, but as we are displaying it as walk distance in the UI it might make sense to name it walk distance? 🤔

Reply:
That was a thought I had after I made this comment too. Can keep it as is.

Reply:
I think that is a good point also. our type is shown as walk distance, so mapping it makes sense to me